compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUTileFuseAndVectorizeLinalgTensorOps.cpp

```diff
@@ -289,6 +280,21 @@ void LLVMCPUTileFuseAndVectorizePass::runOnOperation() {
       return signalPassFailure();
     }
 
+    RewritePatternSet castAwayUnitDimPatterns(&getContext());
+    vector::populateCastAwayVectorLeadingOneDimPatterns(
+        castAwayUnitDimPatterns);
+    if (failed(applyPatternsAndFoldGreedily(
+            funcOp, std::move(castAwayUnitDimPatterns)))) {
+      return signalPassFailure();
+    }
+
+    RewritePatternSet fooPatterns(&getContext());
+    vector::populateVectorReductionToContractPatterns(fooPatterns);
+    vector::ExtractOp::getCanonicalizationPatterns(fooPatterns, context);
+    if (failed(applyPatternsAndFoldGreedily(funcOp, std::move(fooPatterns)))) {
+      return signalPassFailure();
+    }
+
```

Comment:
oh no, I just deleted the file. It might work if you move these to `LLVMCPUAArch64VectorLowering.cpp`.

We put all the optimizations about vector to the pass.

Reply:
I see, thanks. This isn't aarch64-specific, but i can see that `LLVMCPUAArch64VectorLowering.cpp` is full of non-aarch64-specific stuff already anyway.

Question though - it seems that `compiler/src/iree/compiler/Codegen/LLVMCPU/test/tile_fuse_and_vectorize.mlir` is also gone, along with `--iree-llvmcpu-tile-fuse-and-vectorize`. How can I write a test now, exercising such vectorization-related patterns without compiling all the way down to a target executable?

Reply:
self-replying: just found `aarch64_vector_lowering.mlir`. I'll add to that file.

This all looks great (and thanks for asking me for review on #9495 - i realize now i should have reviewed it :-)  - we just have a shallow naming issue here, `aarch64` in a compiler context suggests a stronger tie to this particular architecture than just the casual "it's on aarch64 that we happen to rely on that at the moment" state of this in IREE.